server/db/models/order.js

```diff
@@ -20,4 +19,4 @@ module.exports = function (db) {
 			}
```

Comment:
the way this will look in our views or html:
``` 

<select name="orderStatus">
    <option>Created</option>
    <option>Pending</option>
     <option>Delivered</option>
    <option>Received</option>
</select>
````

And in routes for the post request for new order, you can just send in req.body and it should work fine.